huaweicloud/test/resource_huaweicloud_sms_templates_read.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"huaweicloud-sdk-go-v3/core/auth/basic"
+	sms "huaweicloud-sdk-go-v3/services/sms/v3"
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/sms/v3/model"
+	region "huaweicloud-sdk-go-v3/services/sms/v3/region"
+)
+
+func main() {
+	ak := "OYFWTNGHT3HG0V0MTXBZ"
```

Comment:
pls remove sensitive config
**et env**: 
export HW_ACCESS_KEY="your_ak"
**use it**: 
acceptance.HW_ACCESS_KEY , acceptance.HW_SECRET_KEY,acceptance.HW_REGION_NAME